packages/zowe-explorer/src/uss/USSTree.ts

```diff
@@ -297,7 +297,7 @@ export class USSTree extends ZoweTreeProvider implements IZoweTree<IZoweUSSTreeN
                 return favsForProfile;
             }
             const validationStatus = await Profiles.getInstance().checkCurrentProfile(element.getProfile());
-            if (validationStatus.status === "unverified") {
+            if (SettingsConfig.getDirectValue(globals.SETTINGS_AUTOMATIC_PROFILE_VALIDATION) && validationStatus.status === "unverified") {
```

Comment:
same, initiate search without checking if server is running

Reply:
resolved in [f85c661](https://github.com/zowe/vscode-extension-for-zowe/pull/2461/commits/f85c66102a748b6dc962e84ec38eed2eee59e45c)